singer-io/tap-google-sheets
# TDL-18745 changed to formatted, and made 2 API calls for datetime values

# Description of change
- Non boolean values in boolean columns falls back as string
- Out of range valid datetimes fall back as string
- Currency is typed as string instead of decimal
- Added jitter=None

# Manual QA steps
- Verify that 2 API calls are made 
- Verify Datetime should accept out of range min-max dates
- Verify Currency is typed as string instead of decimal
- Verify Non-boolean values in boolean datatype columns fall back as string
- Verify the request backoffs for more than a minute for 429(Too many requests) error

 
# Risks
 - 
 
# Rollback steps
 - revert this branch
